roles/kubernetes-apps/policy_controller/calico/templates/calico-kube-cr.yml.j2

```diff
@@ -56,6 +56,7 @@ rules:
       - ipreservations
     verbs:
       - list
+      - watch
```

Comment:
Looks like upstream (https://projectcalico.docs.tigera.io/manifests/calico.yaml) separated the ippools and ipreservations and only granted `watch` on ipreservations`; please check against the upstream manifest and align with it.